inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -240,6 +240,11 @@ class XmlSerializer : public ngraph::AttributeVisitor {
                     ngraph::ValueAccessor<double>& adapter) override {
         m_xml_node.append_attribute(name.c_str()).set_value(adapter.get());
     }
+    void on_adapter(const std::string& name,
```

Comment:
I'm nitpicking, sorry :) but the style of line 243 and 244 does not match style presented above or below, I'd suggest matching to what's below as you have similar parameters here.

Reply:
ye, something weird happened with the style of the files. I will address that